manage_breast_screening/notifications/management/commands/send_message_batch.py

```diff
@@ -12,8 +13,10 @@
     Message,
     MessageBatch,
 )
+from manage_breast_screening.notifications.services.queue import Queue
 
 TZ_INFO = ZoneInfo("Europe/London")
+UNRECOVERABLE_FAILURE_CODES = [400, 413]
```

Comment:
I think better to be explicit about the codes we *won't* recover from them, and leave more open the ones we will? 
But I've tested for all the ones listed on [the ticket](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-10446) so we do have some safety check/indication? 🤔